packages/moertel/src/mortar/Moertel_Tolerances.hpp

```diff
@@ -52,7 +52,7 @@ namespace MOERTEL
 {
 
 const double Nodes_Identical_Epsilon = 1.0e-15;
-const double Projection_Length_Epsilon = 1.0e-10;
```

Comment:
Yes, it is.

Few months ago, Moertel, from time to time, tried to project a first 2D element on a second 2D element which has a normal where the normals of both elements were nearly orthogonal.
In such a case, we want to avoid computing this projection as it means that the projection has no sense. Typically, it occurs for elements that are close to each other and the algorithm used by Moertel to restrict the couple of elements for which the projection has to be computed decides that such a projection is required.

To prevent such a case, a test has been introduced:
The normals of both elements are computed and the inner product is computed.
If the magnitude of the value was smaller than 1.0e-10, no projection was computed.
The issue was that this test was absolute: the norm of the normal depended on the element size.

Now, the used test normalizes the norms. This way, the test is no more mesh size dependent.
The value is way higher because this issue was typically seen with very small elements previously. The normalization impacts the tolerance.

Does it make sense?

Reply:
@kliegeois Makes perfect sense -- thanks for explaining!